html_app/components/FACSModelFactory.js

```diff
@@ -64,7 +63,7 @@ scb.components.FACSModelFactory = function scb_components_FACSModelFactory(model
              */
         }
         self.shape_to_data = function (state) {
-            var shape = state.shape;
+            var shape = ('' + state.shape).toLowerCase();
```

Comment:
I think ``String(state.shape).toLowerCase()`` or ``state.shape.toString().toLowerCase()`` would be clearer -- assuming they work. 

On Mar 26, 2015, at 12:37 PM, annagav <notifications@github.com> wrote:

> In html_app/components/FACSModelFactory.js:
> 
> > @@ -64,7 +63,7 @@ scb.components.FACSModelFactory = function scb_components_FACSModelFactory(model
> >               */
> >          }
> >          self.shape_to_data = function (state) {
> > -            var shape = state.shape;
> > +            var shape = ('' + state.shape).toLowerCase();
> That's right. I moved this transformation from the if statement to the declaration.
> 
> —
> Reply to this email directly or view it on GitHub.
>